packages/fxa-shared/subscriptions/types.ts

```diff
@@ -23,13 +26,16 @@ export interface PlanMetadata {
 
 // https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=COPS&title=SP+Tiered+Product+Support#SPTieredProductSupport-MetadataAgreements
 export interface ProductMetadata {
-  productSet?: string | null;
-  productOrder?: number | null;
+  appStoreLink?: string;
```

Comment:
Hmm, this `appStoreLink` and `playStoreLink` don't appear in [the ecosystem docs](https://mozilla.github.io/ecosystem-platform/docs/fxa-engineering/fxa-sub-platform#product-metadata). Probably need to be added there

Reply:
It should definitely be updated, I'm not sure when these options were added and how they are applied though. I'll file an issue to update the ecosystem docs for them.

Reply:
Filed FXA-2349 to update the docs.